spec/luxWrapper.spec.js

```diff
@@ -15,7 +16,7 @@ function getMockReactComponent( options ) {
 			return { foo: "bar" };
 		},
 		render: function() {
-			return React.createElement( "div", { className: "childy" }, "This is a test DIV" );
+			return React.createElement( "div", { displayName: "MOM", className: "childy" }, "This is a test DIV" );
```

Comment:
`displayName: "MOM"` sounds like something I added to be silly....do we still need it? :)